hvantk/utils/llm_interface.py

```
@@ -0,0 +1,513 @@
+
+            result = {
+                "response_text": response.text,
+                "model_used": self.model,
+                "finish_reason": "stop",  # Gemini doesn't provide this explicitly
+                "usage": {"prompt_tokens": -1, "completion_tokens": -1, "total_tokens": -1}  # Not provided by Gemini API
+            }
+
+        elif self.provider == "local":
+            try:
+                import requests
+                import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)